### ISSUE TITLE
Another way of integrating Svelte+Tailwindcss

### ISSUE THREAD

**Issue by guillermin** [matyunya/smelte#43]
Since you are asking for suggestions in your README, just to let you know that I have integrated smelte in my project using the alternative described by @sarioglu [here](https://dev.to/sarioglu/using-svelte-with-tailwindcss-a-better-approach-47ph). It is very similar to how you have it, but there are some differences (no utils.css, for example).

I'm not saying it's better, just that you should have a look and see if you think it is a good approach and apply what you like.

**Comment by Daegalus** [comment]
I gotta say, I was having some errors related to elevation-3 and @apply, this method linked here fixed it and everything started working.

**Comment by matyunya** [comment]
I think this is fairly similar, but I think I'd like to use the component approach for smelte initialization. Let me see what I can do here.

**Comment by matyunya** [comment]
This approach didn't really work out for smelte because it doesn't allow for passing around arbitrary dynamic class values, they get dropped by purge. Of course it would be great to have only one pipeline but right now that would mean dropping `@apply` in component code (which is probably possible) or writing our own preprocessor to collect all the used classes which is a tad harder to achieve but still doable.

**Comment by Daegalus** [comment]
Any easy way for someone to test this? because I think my @applys and such work fine, and i got it to work using the method in the original post.

https://imgur.com/K2LSDNK (ignore the sign in buttons, those aren't skinned yet)

**Comment by matyunya** [comment]
Are you using it with Sapper?

**Comment by Daegalus** [comment]
Nope, just Svelte, tailwindcss, and rollup

**Comment by matyunya** [comment]
Right, it doesn't work with sapper, he's doing it outside svelte preprocessing:
```
    "watch:tailwind": "postcss static/tailwind.css -o static/index.css -w",
    "build:tailwind": "NODE_ENV=production postcss static/tailwind.css -o static/index.css" ,
```
So it looks like we need our own preprocessor/rollup plugin to handle both cases.

**Comment by Daegalus** [comment]
Ah, ok, I didn't think about sapper. My bad

**Comment by matyunya** [comment]
I've spent some more time investigating this and finally got it to work using only svelte preprocessor and it's freaking slow, like 33 seconds to start.

**Comment by matyunya** [comment]
So I went the opposite way, got rid of all `@apply` in the component code (it should really be there anyway), and removed preprocessor at all, basically leaving me with v1 rollup-plugin-smelte, down to 10s startup time.

**Comment by jadbox** [comment]
Any update on this?
